packages/components/src/modal/README.md

```diff
@@ -215,6 +215,12 @@ This property when set to `true` will render a full screen modal.
 -   Required: No
 -   Default: `false`
 
+#### `contentWidth`: `'small' | 'medium' | 'large'`
+
+If this property is added, it will constrain the `max-width` of the modal's contents, preventing it from growing too wide. This prop only applies when `isFullScreen` is `false`.
+
```

Comment:
Missing `Required: No`

Reply:
Yeah I was... testing you. See if you noticed. Yeah. You nailed it, congrats. 😆